src/elements/content-sidebar/AddTaskMenu.js

```diff
@@ -17,11 +17,12 @@ import './AddTaskMenu.scss';
 type Props = {
     isDisabled: boolean,
     onMenuItemClick: (taskType: TaskType) => void,
+    setAddTaskButtonRef?: Function,
```

Comment:
we can type this more strictly, i.e., use `(element: HTMLButtonElement) => void` or similar

Reply:
Yes